scripts/upload_test_results/requirements.txt

```diff
@@ -0,0 +1,6 @@
+certifi==2020.6.20
+configparser==5.0.0
+gitdb==4.0.5
+GitPython==3.1.7
+smmap==3.0.4
+wget==3.2
```

Comment:
is this still needed?

Reply:
> why aren't you using create_pr() from the github module? this works just fine, I just tested again and it works as expected, basically push branch to your fork and submit the PR from there, so there is no additional manual steps.

Could you share your working piece of code for that PR automation?

Reply:
```
#!/usr/bin/env python3

import argparse
from github import Github
from git import Git
import os
import sys
import semantic_version

GH_TOKEN = os.environ.get('GITHUB_TOKEN', None)

def main():
    parser = argparse.ArgumentParser(
        description=__doc__,
        formatter_class=argparse.RawDescriptionHelpFormatter)

    parser.add_argument(
        "-g",
        "--github-repo",
        default="zephyrproject-rtos/test_results",
        help="Github repository (owner/repo)")

    parser.add_argument(
        "-b",
        "--head",
        help="Head of the branch to be submitted as PR")

    args = parser.parse_args()
    if GH_TOKEN:
        g = Github(GH_TOKEN)
    else:
        sys.exit("GITHUB_TOKEN not found in environment. export GITHUB_TOKEN and retry")

    repo = g.get_repo(args.github_repo)

    body = """This is a PR created using a script"""
    pr = repo.create_pull(title="test PR", body=body, head=args.head, base="master")

if __name__ == '__main__':
    main()
```



I then create a branch (readme2), push it to my fork and call this script using: 

```
python3 autopr.py -b nashif:readme2
```